src/receivers.h

```diff
@@ -0,0 +1,91 @@
+/**
+ * @file receivers.h
+ * @author Jeremie Leska <jeremie.leska@6wind.com>
+ * @brief ietf-subscribed-notifications describe receivers for configured
+ * subcribed notification
+ *
+ * @copyright
+ * Copyright 2023 6WIND S.A.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef NP2SRV_RECEIVERS_H_
+#define NP2SRV_RECEIVERS_H_
+
+#include <unyte-udp-notif/unyte_sender.h>
+
+enum receiver_state {
+    RECEIVER_STATE_NONE,
+    RECEIVER_STATE_ACTIVE,
+    RECEIVER_STATE_SUSPENDED,
+    RECEIVER_STATE_CONNECTING,
+    RECEIVER_STATE_DISCONNECTED
```

Comment:
Please use some prefix just like in every other name. You have used `csn`, so use it here as well.